index.js

```diff
@@ -0,0 +1,32 @@
+const { execSync } = require('child_process');
```

Comment:
I don't understand what's going on in this file. Is this the index.clj for the Lambda@Edge?

Reply:
clj?

Now that I'm looking at it again there's no need for me to invoke the js file via the cli using exexSync. I'll change it to a function, import, and call that function.

What's happening in the file is that we're creating the lambda zip that will be used to gate the reports via cloudfront